test/test_hs071.py

```diff
@@ -75,12 +74,11 @@ def test_slsqp_setDV(self):
         """
         Test that setDV works as expected, even with scaling/offset
         """
-        test_name = "hs071_SLSQP_setDV"
-        histFileName = "{}.hst".format(test_name)
+        self.optName = "SLSQP"
```

Comment:
I did it because I didn't want to pass the same variable around everywhere (it's needed in a lot of places). I think it's okay but would like to hear some feedback on this. @sseraj any thoughts?

Yeah `self.opt` created some obscure pickling error, took me a while to figure it out (still not really sure why that was the case). In any case storing the optimizer version was easy enough.

After looking at the code, I will add some more documentation and also set placeholder attributes in the class, otherwise it's unclear what attributes are needed to use `OptTest`.

Reply:
I can see it being clearer to pass as an argument, but I think it's fine as is because you added the docstring and this is just used for testing.